src/main/kotlin/no/nav/tilleggsstonader/sak/utbetaling/tilkjentytelse/domain/AndelTilkjentYtelse.kt

```diff
@@ -63,6 +64,15 @@ enum class TypeAndel {
     UGYLDIG,
 }
 
+fun MålgruppeType.tilTypeAndel(): TypeAndel {
+    return when (this) {
+        MålgruppeType.AAP, MålgruppeType.UFØRETRYGD, MålgruppeType.NEDSATT_ARBEIDSEVNE -> TypeAndel.TILSYN_BARN_AAP
+        MålgruppeType.OVERGANGSSTØNAD -> TypeAndel.TILSYN_BARN_ENSLIG_FORSØRGER
+        MålgruppeType.OMSTILLINGSSTØNAD -> TypeAndel.TILSYN_BARN_ETTERLATTE
+        MålgruppeType.DAGPENGER -> TypeAndel.UGYLDIG
+    }
+}
+
```

Comment:
Litt usikker på om dette er rett sted/fil å legge denne

Reply:
Tenker den burde være i `TilsynBarnBeregnYtelseSteg` fordi den er unik for TilsynBarn, og skal ikke brukes av andre stønader. 
`MålgruppeType.DAGPENGER` burde kaste feil